packages/core-utils/src/base-service.ts

```diff
@@ -34,9 +34,14 @@ export class BaseService<T> {
       return
     }
 
-    this.logger.info('Service is initializing...')
+    this.logger.info('Service is initializing...', {
+      name: this.name,
```

Comment:
Since `this.logger` is initialized like:
```ts
new Logger({ name })
```
do we need to add this context?

Reply:
^good catch! removing